pkgs/applications/editors/vscode/extensions/overrides.nix

```diff
@@ -0,0 +1,301 @@
+{ lib
+, asciidoctor
+, vscode-utils
+, jq
+, moreutils
+, nixpkgs-fmt
+, clojure-lsp
+, callPackage
+, racket-minimal
+, alejandra
+, python3Packages
+, jdk
+, shellcheck
+, fetchurl
+, llvmPackages_8
+, protobuf
+}:
+
+let inherit (lib) maintainers licenses;
+  inherit (vscode-utils) fetchVsixFromVscodeMarketplace buildVscodeMarketplaceExtension;
+
+in
+# note: this doesn't seem to work well with aliases, gonna need help on that
+self: super: {
```

Comment:
@jonringer for many of these packages, there were overrides that were only setting the maintainers list - I don't think `maintainers` makes much sense for auto-generated package definitions so I removed them. [`612fdf0` (#160978)](https://github.com/NixOS/nixpkgs/pull/160978/commits/612fdf081ef7b9e737fa0c4201d5d0c9aea87332)